src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/BlobManager.java

```diff
@@ -84,10 +85,22 @@ public CloudBlobContainer getContainer(String containerName) throws URISyntaxExc
     }
 
     public List<CloudBlobContainer> listInputContainers() {
-        return StreamSupport
+        List<CloudBlobContainer> cloudBlobContainerList = StreamSupport
             .stream(cloudBlobClient.listContainers().spliterator(), false)
             .filter(c -> !c.getName().endsWith(REJECTED_CONTAINER_NAME_SUFFIX))
+            .filter(this::filterBySelectedContainer)
             .collect(toList());
+
+        if (cloudBlobContainerList.isEmpty()) {
+            log.error("NO Blob container found, configuration for selected container: {}", properties.getBlobSelectedContainer());
```

Comment:
Probably it should be "Container not found for configured container name {} ?

Reply:
Also you can verify this error log in output capture ? something similar to https://github.com/hmcts/bulk-scan-processor/blob/d7c42b665edcea8a916976020290f46ad534bd87/src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ErrorNotificationHandlerTest.java#L180-L185

Reply:
added